packages/content/types/index.d.ts

```diff
@@ -52,10 +63,21 @@ declare module 'vue/types/vue' {
 declare module '@nuxt/types/config/hooks' {
   interface NuxtOptionsHooks {
     'content:file:beforeInsert'?: contentFileBeforeInstert;
+    'content:file:beforeParse'?: contentFileBeforeParse;
     content?: {
       file?: {
         beforeInsert?: contentFileBeforeInstert;
+        beforeParse?: contentFileBeforeParse;
       };
     };
   }
 }
+
+declare module '@nuxt/content' {
+  export interface parsers {
```

Comment:
I think interface names should be PascalCase, like so: `export interface Parsers {`

Reply:
I'm not sure about this but I think this should be equal to the import of all parsers. As the parsers folder is not PascalCase this should stay like this.